ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_layers.py

```diff
@@ -1276,3 +1322,25 @@ def test_rfftn(
         axes=axes,
         norm=norm,
     )
+
+
+@handle_test(
+    fn_tree="functional.ivy.experimental.sliding_window",
+    all_args=_sliding_window_helper(),
+    test_with_out=st.just(False),
+    ground_truth_backend="jax",
+)
+def test_sliding_window(*, all_args, test_flags, backend_fw, fn_name, on_device):
```

Comment:
Sadly I can still see some strange errors in the Ci checks. It might be that they didn't get updated for some reason though... Have you tried running the test locally?

Reply:
Yeah, the tests are failing on my side as well